edgexfoundry/edgex-go
# build(snap): upgrade third-party dependencies

This PR upgrades the 3rd party services of the edgexfoundry snap. 

part |  latest stable | current docker | snap | source
-- | -- | -- | -- | --
vault | 1.8.5 | 1.8.4 | ~1.7.2~ 1.8.5 | ~git~ prebuilt archive
consul | 1.10.4 | 1.10.3 | ~1.9.5~ 1.10.4 | prebuilt archive
redis | 6.2.6 | 6.2.6 | ~6.2.4~ 6.2.6 | git
kong | 2.6.0 | 2.5.1 | ~2.4.1~ 2.6.0 | deb archive
kuiper | 1.3.1 | 1.3.1 | ~1.3.0~ 1.3.1 | git
postgres | 10.19 / 13.5 / 14.1 | 13.4 | 10.19 | deb archive


Signed-off-by: Farshid Tavakolizadeh <farshid.tavakolizadeh@canonical.com>







**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>



## Testing Instructions
Checkbox tests pass.
TAF functional tests pass with similar failure rate: 17/501 (4/138 support notification, 1/69 support scheduler, 12/16 system agent)

```
git clone https://github.com/farshidtz/edgex-go.git -b snap-dep-upgrade
cd edgex-go
snapcraft

sudo snap install edgexfoundry_xxx.snap --dangerous
```

Check the logs
Do smoke testing for updates services

## New Dependency Instructions (If applicable)
